datamade/committee-oversight
# Update landing page and add footer

## Overview

Closes #82 and #97. This PR updates the landing page styles and structure to match our [new mockups](https://app.moqups.com/NUARyAts8L/view/page/aa9df7b72). It also introduces a footer. Note that all committee data is dummy! It should look like this:

![screencapture-localhost-8000-2019-10-16-11_54_27](https://user-images.githubusercontent.com/1094243/66941067-ce6bf980-f00b-11e9-942c-64f8332a1d31.png)

## Testing Instructions

* Delete your volume (may not be necessary): `docker-compose down --volumes`
* Start the site back up: `docker-compose up`
* Open a new window and load `hearings.dump`:  
`docker-compose exec postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump`
* Run migrations: `docker-compose exec app python manage.py migrate`
* Load CMS content: `docker-compose run --rm app python manage.py load_cms_content`
* Load sample committee ratings: `docker-compose run --rm app python manage.py load_committeeratings`
* View `http://localhost:8000` and make sure it looks like the screenshot above
* Make sure it looks good on mobile and at various sizes!
